crates/fuel-core/src/graphql_api/api_service.rs

```diff
@@ -296,6 +298,10 @@ where
     ))
 }
 
+fn initialize_query_costs(config: Config) {
+    QUERY_COSTS.get_or_init(|| config.config.costs);
+}
+
```

Comment:
would suggest to move this into the part where the `OnceLock` is defined.

Reply:
Yeah that's better. Will do.

Reply:
Addressed in https://github.com/FuelLabs/fuel-core/pull/2335/commits/58790851f446e19150d1541f850498fe1d0871cd